es6-promise/es6-promise.d.ts

```diff
@@ -51,6 +51,7 @@ declare module Promise {
 	 * Make a promise that rejects to obj. For consistency and debugging (eg stack traces), obj should be an instanceof Error
 	 */
 	function reject(error: any): Promise<any>;
+	function reject<T>(error: T): Promise<T>;
```

Comment:
is this right? Promise's type parameter specify resolve argument.
I tried with `Promise.reject(1).then(v => console.log(v), err => console.error(err));`.